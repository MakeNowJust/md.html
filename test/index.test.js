it('Render index.html', async () => {
  await page.goto('http://localhost:8080/index.html');
  const body = await page.evaluate(() => document.querySelector('.markdown-body').outerHTML);
  expect(body).toMatchSnapshot();
  const title = await page.evaluate(() => document.title);
  expect(title).toBe('index - md.html');
});

it('Render inline.html', async () => {
  await page.goto('http://localhost:8080/inline.html');
  const body = await page.evaluate(() => document.querySelector('.markdown-body').outerHTML);
  expect(body).toMatchSnapshot();
  const title = await page.evaluate(() => document.title);
  expect(title).toBe('inline - md.html');
});

it('Render block.html', async () => {
  await page.goto('http://localhost:8080/block.html');
  const body = await page.evaluate(() => document.querySelector('.markdown-body').outerHTML);
  expect(body).toMatchSnapshot();
  const title = await page.evaluate(() => document.title);
  expect(title).toBe('block - md.html');
});

it('Render footnote.html', async () => {
  await page.goto('http://localhost:8080/footnote.html');
  const body = await page.evaluate(() => document.querySelector('.markdown-body').outerHTML);
  expect(body).toMatchSnapshot();
  const title = await page.evaluate(() => document.title);
  expect(title).toBe('footnote - md.html');
});
