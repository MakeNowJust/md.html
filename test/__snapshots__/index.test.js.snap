// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render block.html 1`] = `
"<div class=\\"markdown-body\\"><h1 id=\\"block\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#block\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>block</h1>
<blockquote>
<p>block quote</p>
</blockquote>
<pre><code>indented code block
</code></pre>
<pre><code>code block
</code></pre>
<pre><code class=\\"hljs language-javascript\\"><span class=\\"hljs-built_in\\">console</span>.log(<span class=\\"hljs-string\\">'code block with syntax highlighting'</span>);
</code></pre>
<ul>
<li>unordered</li>
<li>list</li>
<li>block</li>
</ul>
<ol>
<li>ordered</li>
<li>list</li>
<li>block</li>
</ol>
<table>
<thead>
<tr>
<th>table</th>
<th>block</th>
</tr>
</thead>
<tbody>
<tr>
<td>row</td>
<td>row</td>
</tr>
</tbody>
</table>
<div class=\\"math\\"><span class=\\"katex-display\\"><span class=\\"katex\\"><span class=\\"katex-mathml\\"><math><semantics><mtable><mtr><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mi>e</mi><mo>=</mo></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mrow></mrow><mtext>&nbsp;</mtext><msub><mi>e</mi><mn>1</mn></msub><mo>⋅</mo><msub><mi>e</mi><mn>2</mn></msub></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mo>(</mo><mtext>concatination</mtext><mo>)</mo></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mi mathvariant=\\"normal\\">∣</mi></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mrow></mrow><mtext>&nbsp;</mtext><msub><mi>e</mi><mn>1</mn></msub><mo>∨</mo><msub><mi>e</mi><mn>2</mn></msub></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mo>(</mo><mtext>alternation</mtext><mo>)</mo></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mi mathvariant=\\"normal\\">∣</mi></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mrow></mrow><mtext>&nbsp;</mtext><msup><mi>e</mi><mo>∗</mo></msup></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mo>(</mo><mtext>repeatation</mtext><mo>)</mo></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mi mathvariant=\\"normal\\">∣</mi></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mrow></mrow><mtext>&nbsp;</mtext><mi>c</mi></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mo>(</mo><mtext>character</mtext><mo>)</mo></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mi mathvariant=\\"normal\\">∣</mi></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mrow></mrow><mtext>&nbsp;</mtext><mi>ϵ</mi></mrow></mstyle></mtd><mtd><mstyle scriptlevel=\\"0\\" displaystyle=\\"true\\"><mrow><mo>(</mo><mtext>empty&nbsp;string</mtext><mo>)</mo></mrow></mstyle></mtd></mtr></mtable><annotation encoding=\\"application/x-tex\\">\\\\begin{aligned}
e = &amp;\\\\ e_1 \\\\cdot e_2 &amp; (\\\\text{concatination}) \\\\\\\\
  | &amp;\\\\ e_1 \\\\lor e_2  &amp; (\\\\text{alternation})   \\\\\\\\
  | &amp;\\\\ e^\\\\ast        &amp; (\\\\text{repeatation})   \\\\\\\\
  | &amp;\\\\ c             &amp; (\\\\text{character})     \\\\\\\\
  | &amp;\\\\ \\\\epsilon      &amp; (\\\\text{empty string})
\\\\end{aligned}</annotation></semantics></math></span><span class=\\"katex-html\\" aria-hidden=\\"true\\"><span class=\\"base\\"><span class=\\"strut\\" style=\\"height:7.500000000000002em;vertical-align:-3.5000000000000018em;\\"></span><span class=\\"mord\\"><span class=\\"mtable\\"><span class=\\"col-align-r\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:4em;\\"><span style=\\"top:-6.16em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"mspace\\" style=\\"margin-right:0.2777777777777778em;\\"></span><span class=\\"mrel\\">=</span></span></span><span style=\\"top:-4.659999999999999em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\">∣</span></span></span><span style=\\"top:-3.1599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\">∣</span></span></span><span style=\\"top:-1.6599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\">∣</span></span></span><span style=\\"top:-0.15999999999999837em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\">∣</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:3.5000000000000018em;\\"><span></span></span></span></span></span><span class=\\"col-align-l\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:4em;\\"><span style=\\"top:-6.16em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\"></span><span class=\\"mspace\\">&nbsp;</span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.30110799999999993em;\\"><span style=\\"top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mord mtight\\">1</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.15em;\\"><span></span></span></span></span></span></span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span><span class=\\"mbin\\">⋅</span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.30110799999999993em;\\"><span style=\\"top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mord mtight\\">2</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.15em;\\"><span></span></span></span></span></span></span></span></span><span style=\\"top:-4.659999999999999em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\"></span><span class=\\"mspace\\">&nbsp;</span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.30110799999999993em;\\"><span style=\\"top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mord mtight\\">1</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.15em;\\"><span></span></span></span></span></span></span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span><span class=\\"mbin\\">∨</span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.30110799999999993em;\\"><span style=\\"top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mord mtight\\">2</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.15em;\\"><span></span></span></span></span></span></span></span></span><span style=\\"top:-3.1599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\"></span><span class=\\"mspace\\">&nbsp;</span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.738696em;\\"><span style=\\"top:-3.113em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mbin mtight\\">∗</span></span></span></span></span></span></span></span></span></span><span style=\\"top:-1.6599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\"></span><span class=\\"mspace\\">&nbsp;</span><span class=\\"mord mathdefault\\">c</span></span></span><span style=\\"top:-0.15999999999999837em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mord\\"></span><span class=\\"mspace\\">&nbsp;</span><span class=\\"mord mathdefault\\">ϵ</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:3.5000000000000018em;\\"><span></span></span></span></span></span><span class=\\"arraycolsep\\" style=\\"width:1em;\\"></span><span class=\\"col-align-r\\"><span class=\\"vlist-t vlist-t2\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:4em;\\"><span style=\\"top:-6.16em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mopen\\">(</span><span class=\\"mord text\\"><span class=\\"mord\\">concatination</span></span><span class=\\"mclose\\">)</span></span></span><span style=\\"top:-4.659999999999999em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mopen\\">(</span><span class=\\"mord text\\"><span class=\\"mord\\">alternation</span></span><span class=\\"mclose\\">)</span></span></span><span style=\\"top:-3.1599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mopen\\">(</span><span class=\\"mord text\\"><span class=\\"mord\\">repeatation</span></span><span class=\\"mclose\\">)</span></span></span><span style=\\"top:-1.6599999999999984em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mopen\\">(</span><span class=\\"mord text\\"><span class=\\"mord\\">character</span></span><span class=\\"mclose\\">)</span></span></span><span style=\\"top:-0.15999999999999837em;\\"><span class=\\"pstrut\\" style=\\"height:3em;\\"></span><span class=\\"mord\\"><span class=\\"mopen\\">(</span><span class=\\"mord text\\"><span class=\\"mord\\">empty&nbsp;string</span></span><span class=\\"mclose\\">)</span></span></span></span><span class=\\"vlist-s\\">​</span></span><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:3.5000000000000018em;\\"><span></span></span></span></span></span></span></span></span></span></span></span></div>
<h2 id=\\"h2\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#h2\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>h2</h2>
<h3 id=\\"h3\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#h3\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>h3</h3>
<h4 id=\\"h4\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#h4\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>h4</h4>
<h5 id=\\"h5\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#h5\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>h5</h5>
<h6 id=\\"h6\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#h6\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>h6</h6></div>"
`;

exports[`Render index.html 1`] = `
"<div class=\\"markdown-body\\"><h1 id=\\"-mdhtml\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#-mdhtml\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>📝 md.html</h1>
<ul>
<li><a href=\\"./inline.html\\">inline</a></li>
<li><a href=\\"./block.html\\">block</a></li>
</ul><footer class=\\"md-html-footer\\">Powered by <a target=\\"_blank\\" rel=\\"nofollow noopener noreferrer\\" href=\\"https://github.com/MakeNowJust/md.html/\\">📝 md.html</a></footer></div>"
`;

exports[`Render inline.html 1`] = `
"<div class=\\"markdown-body\\"><h1 id=\\"inline\\"><a class=\\"anchor\\" aria-hidden=\\"\\" href=\\"#inline\\"><svg class=\\"octicon octicon-link\\" viewBox=\\"0 0 16 16\\" version=\\"1.1\\" height=\\"16\\" width=\\"16\\" aria-hidden=\\"aria-hidden\\"><path fill-rule=\\"evenodd\\" d=\\"M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z\\"></path></svg></a>inline</h1>
<p>plain text<br>
break</p>
<p><strong><em>bold italic</em></strong></p>
<p><strong>bold</strong></p>
<p><em>italic</em></p>
<p><del>strikethrough</del></p>
<p><code>inline code</code></p>
<p><span class=\\"inlineMath\\"><span class=\\"katex\\"><span class=\\"katex-mathml\\"><math><semantics><mrow><msup><mi>e</mi><mrow><mi>i</mi><mi>π</mi></mrow></msup><mo>+</mo><mn>1</mn><mo>=</mo><mn>0</mn></mrow><annotation encoding=\\"application/x-tex\\">e^{i\\\\pi} + 1 = 0</annotation></semantics></math></span><span class=\\"katex-html\\" aria-hidden=\\"true\\"><span class=\\"base\\"><span class=\\"strut\\" style=\\"height:0.907994em;vertical-align:-0.08333em;\\"></span><span class=\\"mord\\"><span class=\\"mord mathdefault\\">e</span><span class=\\"msupsub\\"><span class=\\"vlist-t\\"><span class=\\"vlist-r\\"><span class=\\"vlist\\" style=\\"height:0.824664em;\\"><span style=\\"top:-3.063em;margin-right:0.05em;\\"><span class=\\"pstrut\\" style=\\"height:2.7em;\\"></span><span class=\\"sizing reset-size6 size3 mtight\\"><span class=\\"mord mtight\\"><span class=\\"mord mathdefault mtight\\">i</span><span class=\\"mord mathdefault mtight\\" style=\\"margin-right:0.03588em;\\">π</span></span></span></span></span></span></span></span></span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span><span class=\\"mbin\\">+</span><span class=\\"mspace\\" style=\\"margin-right:0.2222222222222222em;\\"></span></span><span class=\\"base\\"><span class=\\"strut\\" style=\\"height:0.64444em;vertical-align:0em;\\"></span><span class=\\"mord\\">1</span><span class=\\"mspace\\" style=\\"margin-right:0.2777777777777778em;\\"></span><span class=\\"mrel\\">=</span><span class=\\"mspace\\" style=\\"margin-right:0.2777777777777778em;\\"></span></span><span class=\\"base\\"><span class=\\"strut\\" style=\\"height:0.64444em;vertical-align:0em;\\"></span><span class=\\"mord\\">0</span></span></span></span></span></p>
<p><a href=\\"./index.html\\">link</a></p>
<p><a href=\\"https://google.com/\\" target=\\"_blank\\" rel=\\"nofollow noopener noreferrer\\">external link</a></p>
<p><img src=\\"https://picsum.photos/300/200\\" alt=\\"image\\"></p></div>"
`;
